newIDE/app/src/GameDashboard/LeaderboardAdmin/index.js

```diff
@@ -483,6 +489,25 @@ export const LeaderboardAdmin = ({ onLoading, project }: Props) => {
       </Line>
     );
 
+  const getExtremeAllowedScoreText = (currentLeaderboard: Leaderboard) => {
```

Comment:
I think you can either:
- Have this function outside the component because it does not use anything that belongs to the component (neither the props nor the hooks)
- Have this function take no input so that it uses current leaderboard from the component, and it will itself do the checks of current leaderboard definition that you're currently doing below